### ISSUE TITLE
Include license files named as LICENCE (UK English)

### ISSUE THREAD

**Issue by likashefqet** [k-tomoyasu/react-native-oss-license#51]
There is one library that has the license file named in UK English (LICENCE) and it is not being included in the licenses
https://dictionary.cambridge.org/dictionary/english/licence

`package.js` example

```json
{
  "name": "template",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "oss-license": "react-native-oss-license --json --only-direct-dependency",
    "start": "react-native start",
    "test": "jest",
    "lint": "eslint ."
  },
  "dependencies": {
    "@react-native-community/cameraroll": "4.0.1",
    "react": "17.0.2",
    "react-native": "0.66.0"
  },
  "devDependencies": {
    "@babel/core": "^7.15.8",
    "@babel/runtime": "^7.15.4",
    "@react-native-community/eslint-config": "^3.0.1",
    "babel-jest": "^27.2.5",
    "eslint": "^8.0.0",
    "jest": "^27.2.5",
    "metro-react-native-babel-preset": "^0.66.2",
    "react-native-oss-license": "^0.4.1",
    "react-test-renderer": "17.0.2"
  },
  "jest": {
    "preset": "react-native"
  }
}
```
`> react-native-oss-license --json --only-direct-dependency`
The result is missing the `_licenseContent` for `@react-native-community/cameraroll` because their license file is named as `LICENCE`
```json
[
    {
        "_description": "React Native Camera Roll for iOS & Android",
        "_license": "MIT",
        "author": {
            "email": "bartol.k@gmail.com",
            "name": "Bartol Karuza"
        },
        "homepage": "https://github.com/react-native-community/react-native-cameraroll#readme",
        "libraryName": "@react-native-community/cameraroll",
        "repository": {
            "type": "git",
            "url": "git+https://github.com/react-native-community/react-native-cameraroll.git"
        },
        "version": "4.0.1"
    },
    {
        "_description": "React is a JavaScript library for building user interfaces.",
        "_license": "MIT",
        "_licenseContent": "MIT License\n\nCopyright (c) Facebook, Inc. and its affiliates.\n\nPermission is hereby granted, free of charge, to any person obtaining a copy\nof this software and associated documentation files (the \"Software\"), to deal\nin the Software without restriction, including without limitation the rights\nto use, copy, modify, merge, publish, distribute, sublicense, and/or sell\ncopies of the Software, and to permit persons to whom the Software is\nfurnished to do so, subject to the following conditions:\n\nThe above copyright notice and this permission notice shall be included in all\ncopies or substantial portions of the Software.\n\nTHE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR\nIMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\nFITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE\nAUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\nLIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,\nOUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE\nSOFTWARE.\n",
        "homepage": "https://reactjs.org/",
        "libraryName": "react",
        "repository": {
            "directory": "packages/react",
            "type": "git",
            "url": "git+https://github.com/facebook/react.git"
        },
        "version": "17.0.2"
    },
    {
        "_description": "A framework for building native apps using React",
        "_license": "MIT",
        "_licenseContent": "MIT License\n\nCopyright (c) Facebook, Inc. and its affiliates.\n\nPermission is hereby granted, free of charge, to any person obtaining a copy\nof this software and associated documentation files (the \"Software\"), to deal\nin the Software without restriction, including without limitation the rights\nto use, copy, modify, merge, publish, distribute, sublicense, and/or sell\ncopies of the Software, and to permit persons to whom the Software is\nfurnished to do so, subject to the following conditions:\n\nThe above copyright notice and this permission notice shall be included in all\ncopies or substantial portions of the Software.\n\nTHE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR\nIMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\nFITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE\nAUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\nLIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,\nOUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE\nSOFTWARE.\n",
        "homepage": "https://github.com/facebook/react-native#readme",
        "libraryName": "react-native",
        "repository": {
            "type": "git",
            "url": "git+ssh://git@github.com/facebook/react-native.git"
        },
        "version": "0.66.0"
    }
]
```

**Comment by likashefqet** [comment]
**Suggestion**
https://github.com/k-tomoyasu/react-native-oss-license/blob/f2a29b5a7e99a6b3f8ec41d606a6fc6986c33c6e/src/reader/DependencyReader.ts#L38
```typescript
      path.join(pkg.path, '{LICEN[SC]E,[Ll]icen[sc]e}*')
```

**Comment by k-tomoyasu** [comment]
Thanks for your suggestion!
It looks good 👍 
I'm going to fix it, or if you could send  the pull request I will gladly accept it!

**Comment by k-tomoyasu** [comment]
I have published your patch as version 0.5.1.
Thanks 👍
